src/components/backup/BackupSheetKeyboardLayout.js

```diff
@@ -11,7 +11,7 @@ import styled from '@/styled-thing';
 import { padding } from '@/styles';
 
 const Footer = styled(Column)(({ isTallPhone }) => ({
-  ...padding.object(20, 15, isTallPhone ? 30 : 15),
+  ...padding.object(20, 15, isTallPhone ? 65 : 50),
```

Comment:
i fixed the `RestoreSheet` height to match our other sheets, as a result I had to adjust the padding of a button so it didn't get clipped by the keyboard